appinventor/blocklyeditor/src/blocklyeditor.js

```diff
@@ -514,79 +1085,128 @@ Blockly.ai_inject = function(container, workspace) {
         }
       }
     };
-    workspace.addChangeListener(workspaceChanged);
+    //workspace.addChangeListener(workspaceChanged);
```

Comment:
Figure out whether this still needs to be commented out.

Reply:
I got scared when I saw this because I didn't remember the changes, but then I saw that it wasn't me that made the change ;-)